pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -145,24 +146,43 @@ def _configure_checkpoint_callbacks(self, checkpoint_callback: bool) -> None:
         if not self._trainer_has_checkpoint_callbacks() and checkpoint_callback is True:
             self.trainer.callbacks.append(ModelCheckpoint())
 
-    def _configure_model_summary_callback(self, weights_summary: Optional[str] = None) -> None:
+    def _configure_model_summary_callback(
+        self, enable_model_summary: bool, weights_summary: Optional[str] = None
+    ) -> None:
+        if not enable_model_summary:
+            return
         if any(isinstance(cb, ModelSummary) for cb in self.trainer.callbacks):
             return
-        if weights_summary is not None:
+        if weights_summary is None:
+            rank_zero_deprecation(
+                "Setting `Trainer(weights_summary=None)` is deprecated in v1.5 and will be removed"
+                " in v1.7. Please set `Trainer(enable_model_summary=False) instead."
+            )
+            return
+        # Prior default in the Trainer for `weights_summary` which we explicitly check here
+        # to preserve backwards compatibility
+        if weights_summary != "top":
+            rank_zero_deprecation(
+                f"Setting `Trainer(weights_summary={weights_summary})` is deprecated in v1.5 and will be removed"
+                " in v1.7. Please pass `pytorch_lightning.callbacks.model_summary.ModelSummary` with"
+                " `max_depth` directly to the Trainer's `callbacks` argument instead."
```

Comment:
```suggestion
                f"Setting `Trainer(weights_summary={weights_summary})` is deprecated in v1.5 and will be removed "
                "in v1.7. Please pass `pytorch_lightning.callbacks.model_summary.ModelSummary` with "
                "`max_depth` directly to the Trainer's `callbacks` argument instead."
```
same here.

Reply:
in this file it looks like the spaces are all prepended to the following line besides the gradient accumulation error message

Reply:
all the new ones are actually following this one (spaces appended at the end)